src/commands/suggest.ts

```diff
@@ -0,0 +1,41 @@
+// Codey suggest Command
+import Discord from 'discord.js';
+import { openDB } from '../components/db';
+
+export const suggestCmd = async (message: Discord.Message, args: string[]) => {
+  const db = openDB();
+  const state = 'C'; // Create state = C
+  const helpArg = 'help ';
+  let words = '';
+  let word = '';
+  // Turn args into suggestion
+  for (word in args) {
+    words += args[word] + ' ';
+  }
+
+  if (words == '') {
+    message.channel.send('Codey sees an empty suggestion! Try again.');
+  } else if (words == helpArg) {
+    message.channel.send(
+      '.suggest <suggestion> \nYour <suggestion> should only contain text and is capped at about 100 words.'
+    );
+  } else {
+    try {
+      // Save suggestion into DB
+      (
+        await db
+      ).run(
+        'INSERT INTO suggestions (suggestion_id, suggestion_author, suggestion, suggestion_state) VALUES(?,?,?,?);',
+        [null, message.author.id, words, state]
+      );
+
+      // Confirm suggestion was taken
+      message.channel.send('Codey has recieved your suggestion.');
+    } catch (err) {
```

Comment:
i don't think we need to try catch here if we have try catch in the handleCommand function, since this will prevent the app from logging errors, because we can send the error message in that error handler instead.